### DISCUSSION TITLE
some questions and suggestions

### DISCUSSION THREAD

**Discussion by ghost** [filecxx/FileCentipede#568] [Ideas]
first off, it's a great client. I've tested it with a 40 GB torrent. it was instantly recognized by the client and download began in an instant too. connected in the speed of light. respected the speed limit, which was set to 15mb, with impressive accuracy - qbittorrent, on the other hand, not only ignores your limit but also has a notoriously inconsistent download speed. I greatly appreciate it that you enabled sequential downloads as an option; my hard drive gets really noisy when it is non-sequential (I understand the issues with it, but my drives are LOUD so I only disable sequential when I'm not home).

the client is a bit heavier than utorrent 2.2.1, which consumes only 6mb ram on idle, but that's partly due to the fact utorrent 2.2.1 is 32 bit. File Centipede consumes approximately 19.4MB of ram on idle, being beaten by qbittorrent by a few hundred kilobytes. if you consider file centipede has at least 5 functions qbit hasn't, it surely is an incredibly compact memory footprint. 

regarding the UI, it's simply the best one out there. doesn't look bland like transmission, nor outdated like utorrent. qbit's UI used to be good till they fucked it all up with those horrid abominations of icons. C++ code makes it snappy, lightweight, performant, unlike some Java and Python clients out there.

features lacking, in order of importance:

1 - pre-allocation of torrent files on disk 
2 - recheck torrent on completion
3 - network adapter binding
4 - tags
5 - options to enable/disable PeX, DHT
6 - advanced torrent options
7 - enable/disable UDP, TCP
8 - (cosmetic) ability to show a list of trackers and their favicons

that's pretty much it for the lacking part. pre-allocation is useful for those of us running HDD's.

other suggestions (in no order):

1 - (cosmetic) either add some skins or make it so that the community can create some :)
2 - there are many features, and that is great, but avoid over-extending yourself
3 - keep it up with the Windows 7/Vista support. that is really cool.
4 - create an appimage for linux users so that they can all use it regardless of distribution

When people find out about this client, I'm pretty sure qbit will be gone in no time. that thing is buggy as hell and the devs made it unbearably ugly in the last versions (and removed win7 compatibility with qt6). thanks for the program, keep up the good work, it will be worth it

by the way, I made a post on r/torrents asking people what did they think about your client. my post got deleted by mods after a few minutes, with no explanation. weird.

**Comment by filecxx** [comment]
Thanks for your support. 
I want to assure you that all these missing features will be added in the upcoming FileCentipede 3.0. Also, I promise that support for Windows 7 and Vista will never be dropped.

**Reply by ghost** [reply]
Hi, one more thing: please add some more status options on the torrent status bar. Ratio and availability, to be more specific

**Reply by filecxx** [reply]
No problem. context menu of columns already on the schedule.
